frontend/src/pages/chartTest.js

```diff
@@ -0,0 +1,117 @@
+import React, { useEffect, useRef, useState } from 'react';
+import Chart from 'chart.js';
+import Button from 'react-bootstrap/Button';
+import 'bootstrap/dist/css/bootstrap.min.css';
+import Container from 'react-bootstrap/Container';
+import Row from 'react-bootstrap/Row';
+import Col from 'react-bootstrap/Col';
+
+import Radar from './chart/chart';
+
+
+function RadarTest(props) {
+
+    const [input, setInput] = useState(0);
+    const [testInput1, settestInput1] = useState(0);
+    const [testInput2, settestInput2] = useState(0);
+    const [testInput3, settestInput3] = useState(0);
+
+
+    const scenario_scoring_matrix = {
+        Tim: {
+            issue1: 1,
+            issue2: 4,
+            issue3: 2
+        },
+        Song: {
+            issue1: 0,
+            issue2: 2,
+            issue3: 3
+        }
+
+    };
+
+    const Total_Emphasis = {};
```

Comment:
this section is pretty confusing, and will be hard to test as is. It should be broken up into methods that can be called so they methods can be tested, and it will make the code more readable.